.github/workflows/ci.yaml

```diff
@@ -15,25 +15,26 @@ jobs:
       fail-fast: false
       matrix:
         # different operating systems
-        os: [ubuntu-18.04, macos-10.15, windows-2019]
-        features: [--features default]
+        os: [ubuntu-18.04, ubuntu-20.04, macos-10.15, windows-2019]
```

Comment:
maybe we should start migrating off 18.04?

Reply:
I would agree, but they're still using 18.04 as `ubuntu-latest`, so I would say we should wait until they change that before we use 20.04 as our primary OS in the matrix.